prow/cmd/jenkins-operator/main.go

```diff
@@ -91,8 +96,16 @@ func main() {
 		ac.BearerToken = &jenkins.BearerTokenAuthConfig{
 			Token: token,
 		}
+	} else if *certFile != "" && *keyFile != "" && *caCertFile != "" {
```

Comment:
@dobbymoodge I have a feeling that the certs are complementary to doing basic auth. Is that correct?